docs/source/guide/lre-5-theory.md

```diff
@@ -0,0 +1,85 @@
+on unitary folding, go to [What is the theory behind ZNE?](zne-5-theory.md).
+
+Suppose we want to estimate the noiseless expectation value of some observable in an $n$-qubit circuit with $l$ layers.
+
+Each layer can have a different scale factor and we can create $M$ such variations of the scaled circuit. Let $\{λ_1, λ_2, λ_3, \ldots, λ_M\}$ be the scale factors vectors used to create multiple variations of the noise-scaled circuits $\{C_{λ_1}, C_{λ_2}, C_{λ_3}, \ldots, C_{λ_M}\}$ such that each vector $λ_i$ defines the scale factors for the different layers in the input circuit $\{{λ^1}_i, {λ^2}_i, {λ^3}_i, \ldots, {λ^l}_i\}^T$.
+
+If $d$ is the chosen degree of our multivariate polynomial, $M_j(λ_i, d)$ corresponds to the terms in the polynomial. In general, the monomial terms for a variable $l$ up to degree $d$ can be determined through the [stars and bars method](https://en.wikipedia.org/wiki/Stars_and_bars_%28combinatorics%29).
+
+$$
+\text{total number of terms in the monomial basis with max degree } d = \binom{d + l}{d}
+$$
+
+$$
+\text{number of terms in the monomial basis with total degree } d = \binom{d + l - 1}{d}
+$$
+
+These monomial terms define the rows of the square sample matrix where $a_{i,j}=M_j(λ_i, d)$.
+
+$$
+\mathbf{A}(\Lambda, d) = 
+\begin{bmatrix}
+    a_{1,1} & a_{1,2} & \cdots & a_{1,M} \\
+    a_{2,1} & a_{2,2} & \cdots & a_{2,M} \\
+    \vdots & \vdots & \ddots & \vdots \\
+    a_{N,1} & a_{N,2} & \cdots & a_{N,M}
+\end{bmatrix}
+$$
+
+Each monomial term in the sample matrix is evaluated using the values in the scale factor vectors. We aim to define the zero-noise limit as a linear combination of the noisy expectation values. Finding the coefficients in the linear combination becomes a problem solvable through a system of linear equations $Ac = z$ where $c$ is the coefficients vector, $z$ is the vector of expectation values and $\mathbf{A}$ is the sample matrix evaluated using the values in the scale factor vectors.
+
+## Step 2: Extrapolate to the noiseless limit
+
+Each noise scaled circuit $C_{λ_i}$ has an expectation value associated with it $\langle O(λ_i) \rangle$ such that we can define a vector of the noisy expectation values $z = (\langle O(λ_1) \rangle, \langle O(λ_2) \rangle, \langle O(λ_3) \rangle, \ldots, \langle O(λ_M)\rangle)^T$. These have a coefficient of linear combination associated with them such that 
+
+$$
+O_{LRE} = \sum_{i=1}^{M} \eta_i \langle O(\boldsymbol{\lambda}_i) \rangle.
+$$
+
+The system of linear equations is used to find the numerous $\eta_i$. As we only need to find the noiseless expectation value, we do not need to calculate the full vector of linear combination coefficients if we use the [Lagrange interpolation formula](https://files.eric.ed.gov/fulltext/EJ1231189.pdf). 
+
+$$
+O_{\rm LRE} = \sum_{i=1}^M \langle O (\boldsymbol{\lambda}_i)\rangle  \frac{\det \left(\mathbf{M}_i (\boldsymbol{0}) \right)}{\det \left(\mathbf{A}\right)}.
```

Comment:
This is supposed to be one of the terms in the monomial basis evaluated at $\lambda=0$

Reply:
Why not just use $M_i(\boldsymbol{0}, d)$? That's the notation defined above. Also since we defined $M$ with a second argument, I don't think we should drop it.